src/app/pages/details/details.js

```diff
@@ -289,6 +289,7 @@ export default class Details extends BaseClass {
             detailsTab.emit('put-toc-in', this.regionFrom(this.el.querySelector('.toc-slideout-contents')));
             if (!polish && this.pageData.free_stuff_instructor.content) {
```

Comment:
if not Polish? so this doesn't track Polish books?

Reply:
There are no instructor resources for Polish books.